docs/data/external-oasis-repos.md

```diff
@@ -179,7 +179,9 @@ In order to host a repository on OASIS, perform the following steps:
     representative before continuing with the remaining instructions; for all other repositories (such as `*.egi.eu`),
     you are done.
 
-1. One you are told in the ticket to proceed to the next step, execute the following commands:
+1. When you are told in the ticket to proceed to the next step, make sure that
+    the repository is not in a transaction (use `cvmfs_server abort` if not
+    sure) and execute the following commands:
```

Comment:
Could we separate the `cvmfs_server abort` bit into its own step? It seems logically separate enough. Also, is there a command that can be run to detect if the repo is in a transaction so they can run the `abort` based on that information?

Reply:
Unfortunately there's not a straightforward way to detect if a repo is in transaction.  The abort could be in a separate step, I guess I'll do that.